examples/validation/validate_skybox.py

```diff
@@ -0,0 +1,63 @@
+"""
+Validate Skybox
```

Comment:
@panxinmiao I invite you to make a suggestion somewhere. That way your name gets attached to the commit for this PR, and you'd get the credits/stats for your work.

Reply:
Thank you, you are so considerate, 😄

Reply:
Only trying to be fair. I initially wanted to cherry-pick a commit from your PR, but everything has evolved too far to make that work :)